neqo-transport/src/connection/tests/priority.rs

```diff
@@ -78,10 +81,60 @@ fn relative() {
     let dgram = client.process_output(now()).dgram();
     server.process_input(dgram.unwrap(), now());
 
-    // The "normal" stream will get a `NewStream` event, but no data.
+    // The "id_normal" stream will get a `NewStream` event, but no data.
     for e in server.events() {
         if let ConnectionEvent::RecvStreamReadable { stream_id } = e {
-            assert_ne!(stream_id, normal);
+            assert_ne!(stream_id, id_normal);
+        }
+    }
+}
+
+/// Higher priority streams get sent ahead of lower ones, even when
+/// the higher priority stream is written to later.
```

Comment:
Please update this comment. It is the same as in the previous test, but something else is tested here.